pkg/register/documentBuilder.go

```diff
@@ -82,19 +82,14 @@ func (b RegisterDocumentBuilder) build() (*RegisterDocument, error) {
 		specVersion = b.SpecVersion
 	}
 
-	updateTime := defaultUpdateTimeSeconds()
-	if b.UpdateTime != 0 {
-		updateTime = b.UpdateTime
```

Comment:
`RegisterDocumentBuilder.UpdateTime` can now be removed?

Reply:
It cannot be removed as RegisterDocument is used to marshal/unmarshal the document json.